ui/package.json

```
@@ -19,7 +19,7 @@
     "@mui/icons-material": "^6.4.12",
     "@mui/lab": "6.0.0-beta.19",
     "@mui/material": "^6.4.12",
-    "@yokowu/modelkit-ui": "2.0.6",
+    "@yokowu/modelkit-ui": "2.3.1",
```

UI 包的版本更新也需要验证是否引入了破坏性变更。确认 @yokowu/modelkit-ui 从 2.0.6 到 2.3.1 的更新不会影响现有功能。

```suggestion
// 检查 @yokowu/modelkit-ui 的更新日志
// 确保没有破坏现有功能